e2e/vertical-full-page-map.spec.ts

```diff
@@ -81,16 +99,20 @@ test.describe('full page map with filters test suite', () => {
   });
 
   test('clicking on a cluster causes the map to zoom in', async ({ page }) => {
-    const mapboxPinCount = await page.locator('#js-answersMap div').count();
+    const originalCount = await page.locator('.yxt-Card').count();
     await page.getByRole('button', { name: 'Cluster of 2 results' }).click();
-    const mapboxPinCountAfterSelectingCluster = await page.locator('#js-answersMap div').count();
-    expect(mapboxPinCount).not.toBe(mapboxPinCountAfterSelectingCluster);
+     const response = await page.waitForResponse(resp => resp.url().includes('https://prod-cdn.us.yextapis.com/v2/accounts/me/search/vertical/query')
+      && resp.url().includes('input=virginia') 
+      && resp.url().includes('filters'));
+     await expect(response.status()).toBe(200);
+    const countAfterSelectingCluster = await page.locator('.yxt-Card').count();
+    expect(originalCount).toBeGreaterThan(countAfterSelectingCluster);
   });
 
   test('clicking on a cluster causes a new search to be run', async ({ page }) => {
```

Comment:
it looks like we already test this in the test above? if we still want it as a separate test, can we update to check that the response below has a filter so we know it's not matching the original `virginia` query?